srcpkgs/pgbackrest/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'pgbackrest'
+pkgname=pgbackrest
+version=2.15.1
+revision=1
+wrksrc="${pkgname}-release-${version}"
+build_wrksrc="src"
+build_style=gnu-configure
+hostmakedepends="perl"
+makedepends="libressl-devel libxml2-devel perl zlib-devel"
+short_desc="Reliable PostgreSQL Backup & Restore"
+maintainer="Anachron <gith@cron.world>"
+license="MIT"
+homepage="http://www.pgbackrest.org/"
+distfiles="https://github.com/pgbackrest/pgbackrest/archive/release/${version}.tar.gz"
+checksum=e559de3525129ed692ba97c94bd8ea5c86966f7e172c20b64fc95651225bc525
+nocross=yes
```

Comment:
please give an explanation here why it can't be crosscompiled.

Reply:
Short version: it calls into perl during configure to retrieve some cflags/ldflags that are tied to architecture, resulting in either failure or miscompilation

Reply:
that should be reflected (with fewer words of course) in the comment in the template so people interested in getting the package to cross-compile later will know where to look.